docs/guide/contributing.md

```diff
@@ -214,6 +214,27 @@ We cannot control the `html` base font size and therefore these relative units a
 
 TODO: wip
 
+#### Attributes
```

Comment:
docs: can you remove the "TODO: wip" above?

Reply:
Wrote a paragraph about prop naming instead